dbt/include/firebolt/macros/utils/position.sql

```diff
@@ -0,0 +1,3 @@
+{% macro default__position(substring_text, string_text) %}
```

Comment:
why we have `firebold__` prefix everywhere and `default__` here?

Reply:
Bad copy-paste. Should be firebolt_ . default works as well since DBT goes to default if it doesn't find <connector>_ prefixed macro, but overriding it is a bad form as I understand. Will fix it.